extensions/amp-analytics/0.1/vendors/captainmetrics.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {jsonLiteral} from '../../../../src/json';
+
+const CAPTAINMETRICS_CONFIG = jsonLiteral({
+  'transport': {
+    'beacon': true,
+    'xhrpost': true,
+    'useBody': true,
+    'image': false,
+  },
+  'vars': {
+    'hostDomain': 'photon.captainmetrics.com',
```

Comment:
Will it ever be the case where `hostDomain` contains `/` or `?`. Asking because AMP encode variables by default to make sure they don't break the url.

Reply:
Yes this variable will only contain a domain and not a path. We use it to customize the endpoint for our customers and protect against adblockers.

Reply:
Sounds good. FYI we also have `requestOrigin` if you ever want to customize the request's origin field. value from that field won't be encoded.